### PR TITLE
Remove requirement for github repo and team ids

### PR DESCRIPTION
**Pull request by jan-opal**
Removes the requirement for GitHub repos and teams to provide the numeric id.

### CODE REVIEW

File: opal/group_remote_info.go [hunk 1]
```diff
@@ -7,10 +7,11 @@ import (
 )
 
 // NOTE: Unfortunately, terraform go-sdk does not support nested object types natively. The only work-around
-//       is to have a schema.ListType with MaxItems=1 as a layer of indirection in between each level of nesting.
-//       This makes the implementation ugly, but it's thankfully mostly hidden from the client. If nested objects
-//       are ever natively supported it in the SDK, we should be able to update our code without a need for
-//       change in the HCL of clients.
+//
+//	is to have a schema.ListType with MaxItems=1 as a layer of indirection in between each level of nesting.
```
Comment (stephenwan-opal) [inline opal/group_remote_info.go:11]: this diff looks unintentional?

File: opal/resource_remote_info.go [hunk 2]
```diff
@@ -7,10 +7,11 @@ import (
 )
 
 // NOTE: Unfortunately, terraform go-sdk does not support nested object types natively. The only work-around
-//       is to have a schema.ListType with MaxItems=1 as a layer of indirection in between each level of nesting.
-//       This makes the implementation ugly, but it's thankfully mostly hidden from the client. If nested objects
-//       are ever natively supported it in the SDK, we should be able to update our code without a need for
-//       change in the HCL of clients.
+//
+//	is to have a schema.ListType with MaxItems=1 as a layer of indirection in between each level of nesting.
```
Comment (stephenwan-opal) [inline opal/resource_remote_info.go:11]: same note - is the newline intentional?

**Review by stephenwan-opal — APPROVED** [review]
lgtm - would suggest we wait on #4 and re-run tests just to double check

File: opal/group_remote_info.go (hunk 1 above)
Comment (jan-opal) [reply opal/group_remote_info.go]: For some reason go fmt did not like how I commented this in the first place and keeps changing it to that. I changed it to something more sane.

**Pull request merged**